<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uyeislemleri</class>
 <widget class="QDialog" name="uyeislemleri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tw_uyeler">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>351</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Tüm Üyeler</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_yeniKayit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Yeni Kayıt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_guncelle">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>320</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Güncelle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sil">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sil</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>ÜYE İŞLEMLERİ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_uyeSoyad">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>133</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_uyeAd">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>161</y>
     <width>133</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_uyeNo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>133</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>69</width>
     <height>53</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Üye Ad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>69</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Üye Soyad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>69</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Üye No:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
